<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Standard_DialogsClass</class>
 <widget class="QWidget" name="Standard_DialogsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Standard_Dialogs</string>
  </property>
  <widget class="QTabWidget" name="process">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>701</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab0">
    <attribute name="title">
     <string>输入对话框</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>521</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="getDouble">
        <property name="text">
         <string>getDouble</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="getText">
        <property name="text">
         <string>getText</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_input">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="getInt">
        <property name="text">
         <string>getInt</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="getItem">
        <property name="text">
         <string>getItem</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="getMultiText">
        <property name="text">
         <string>getMultiText</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>颜色对话框</string>
    </attribute>
    <widget class="QPushButton" name="getColor">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>getColor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_alpha">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>是否显示透明度</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>字体对话框</string>
    </attribute>
    <widget class="QPushButton" name="getFont">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>getFont</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_font">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>你好，Qt</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>文件对话框</string>
    </attribute>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>601</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="getExistingDirectory">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>getExistingDirectory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getOpenFileName">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileName</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getOpenFileNames">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileNames</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getSaveFileName">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>getSaveFileName</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab4">
    <attribute name="title">
     <string>消息对话框</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="critical">
        <property name="text">
         <string>critical</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="warning">
        <property name="text">
         <string>warning</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="information">
        <property name="text">
         <string>information</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="question">
        <property name="text">
         <string>question</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="showMessage">
        <property name="text">
         <string>showMessage</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab5">
    <attribute name="title">
     <string>进度对话框</string>
    </attribute>
    <widget class="QPushButton" name="progress">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>progress</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab6">
    <attribute name="title">
     <string>向导对话框</string>
    </attribute>
    <widget class="QPushButton" name="wizard">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>wizard</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Standard_Dialogs.qrc"/>
 </resources>
 <connections/>
</ui>
